go/kbfs/kbfsgit/git-remote-keybase/main.go

```diff
@@ -123,8 +124,18 @@ func start() (startErr *libfs.Error) {
 
 	remote := flag.Arg(0)
 	var repo string
+	lfs := false
 	if len(flag.Args()) > 1 {
 		repo = flag.Arg(1)
+	} else {
+		// For LFS invocation, the arguments actually come together in
+		// a single quoted argument for some reason.
+		s := strings.Split(remote, " ")
```

Comment:
maybe use `strings.Fields` instead so it covers different whitespaces, and multiple whitespaces in a row?

Reply:
Sure, I will do that in a follow-on PR I need to do for HOTPOT-1297 anyway.